ros_buildfarm/templates/release/binarydeb_create_task.Dockerfile.em

```diff
@@ -55,6 +55,11 @@ RUN echo "@today_str"
     os_name=os_name,
     os_code_name=os_code_name,
 ))@
+@(TEMPLATE(
+    'snippet/install_dh-python.Dockerfile.em',
+    os_name=os_name,
+    os_code_name=os_code_name,
+))@
```

Comment:
Can you elaborate why the package is needed during the create task docker?

Reply:
I haven't tested it, but I don't think it's needed here as nothing during this task is actually invoking debhelper tasks.